src/components/graph-view.js

```diff
@@ -1295,6 +1311,57 @@ class GraphView extends React.Component<IGraphViewProps, IGraphViewState> {
     ReactDOM.render(anyElement, nodeContainer);
   }
 
+  asyncElevateNodeAndEdges(node: INode) {
+    const asyncElevateNode = node =>
+      requestAnimationFrame(() => this.syncElevateNode(node));
+    const asyncElevateEdge = edge =>
```

Comment:
This can be made api functions too

Reply:
No one else seemed to require this so I kept it in here.